src/harp_pedal_wizard.lua

```diff
@@ -0,0 +1,1591 @@
+    local ret_table = {}
+    for element in val_string:gmatch('[^,%s]+') do  -- lua pattern magic taken from the Internet
+        local parsed_element = parse_parameter(element)
+        table.insert(ret_table, parsed_element)
+    end
+    return ret_table
+end
+
+parse_parameter = function(val_string)
+    if '"' == val_string:sub(1,1) and '"' == val_string:sub(#val_string,#val_string) then -- double-quote string
+        return string.gsub(val_string, '"(.+)"', "%1") -- lua pattern magic: "(.+)" matches all characters between two double-quote marks (no escape chars)
+    elseif "'" == val_string:sub(1,1) and "'" == val_string:sub(#val_string,#val_string) then -- single-quote string
+        return string.gsub(val_string, "'(.+)'", "%1") -- lua pattern magic: '(.+)' matches all characters between two single-quote marks (no escape chars)
+    elseif "{" == val_string:sub(1,1) and "}" == val_string:sub(#val_string,#val_string) then
+        return parse_table(string.gsub(val_string, "{(.+)}", "%1"))
+    elseif "true" == val_string then
+        return true
+    elseif "false" == val_string then
+        return false
+    end
+--    return tonumber(val_string)
+    return val_string
+end
+
+local get_parameters_from_file = function(file_name) -- modified
+    local parameters = {}
+    for line in io.lines(file_name) do
+        local comment_marker = "--"
+        local parameter_delimiter = "="
+        local comment_at = string.find(line, comment_marker, 1, true) -- true means find raw string rather than lua pattern
+        if nil ~= comment_at then
+            line = string.sub(line, 1, comment_at-1)
+        end
+        local delimiter_at = string.find(line, parameter_delimiter, 1, true)
+        if nil ~= delimiter_at then
+            local name = strip_leading_trailing_whitespace(string.sub(line, 1, delimiter_at-1))
+            local val_string = strip_leading_trailing_whitespace(string.sub(line, delimiter_at+1))
+            parameters[name] = parse_parameter(val_string)
+        end
+    end
+    return parameters
+end
+
+-- Modified from library
+function get_parameters(file_name, parameter_list)
+    local file_parameters = get_parameters_from_file(file_name)
+    if nil ~= file_parameters then
+        for param_name, def_val in pairs(file_parameters) do
+            local param_val = file_parameters[param_name]
+            if nil ~= param_val then
+                parameter_list[param_name] = param_val
+            end
+        end
+    end
+    return parameter_list
+end
+
+--------------------------------------
+function path_set(filename)
+    local path = finale.FCString()
+    local path_delimiter = finale.FCString()
+    local ui = finenv.UI()
+    path:SetUserOptionsPath()
+    if ui:IsOnMac() then
+        path_delimiter.LuaString = "/"
+    elseif ui:IsOnWindows() then
+        path_delimiter.LuaString = "\\"
+        --path_delimiter.LuaString = "/" -- apparently Windows can use either! Go figure!
+    end
```

Comment:
Window recognizes "/" as a path delimiter. (This is a lesser-known fact.) But if you want to explicitly use "\\" instead, at least remove the comment.